packages/analytics/src/api.ts

```diff
@@ -224,6 +225,23 @@ export function setAnalyticsCollectionEnabled(
     enabled
   ).catch(e => logger.error(e));
 }
+
+/**
+ * Adds data that will be set on every event logged from the SDK, including automatic ones.
+ * By using gtag's "set" command, the values passed persist on the current page and are passed with
```

Comment:
Strictly speaking the subject of "using" can't really be "the values" -- a correct form would be more like "By using . . . you can ensure that the values passed persist . . ."

I'd suggest "With gtag's set command, the values . . ."